templates/swagger/v1_json.tmpl

```diff
@@ -459,6 +460,7 @@
         "operationId": "orgCreate",
         "parameters": [
           {
+            "description": "'visibility' can be \"public\", \"limited\" or \"private\"",
```

Comment:
As below, the description should describe the body in total and should be in the referenced schema

```suggestion
```